### ISSUE TITLE
hyper params experiment summary

### ISSUE THREAD

**Issue by yzbx** [ISCAS007/torchseg#29]
# hyper params experiment summary

no influence-> obviously influence
:no_mouth: < :heavy_plus_sign: < :yum: < :star:
### semantic segmentation miou (mean/max)
| hyper params | value |  miou |  experiments |  others |
| - | - | - | - | - |
| lr for sgd | 0.01 | - | hyperopt002, hyperopt | :star: 1e-2 |
| lr for adam | 1e-4 | - | hyperopt005/012/013/014 | :star: 1e-4 |
| norm_ways | - | - | hyperopt004, hyperopt007 | :heavy_plus_sign: caffe/cityscapes |
| l2_reg | [1e-5, 0.1] | - | hyperopt006 | :no_mouth: |
| dataset size | 32->320->640->full | - | - | :star: |
| data augmentaion | T/F | - | hyperopt008 | :no_mouth: |
| backbone freeze | T/F | - | hyperopt009 | :heavy_plus_sign: T for datasize=32|
| backbone pretrained | T/F | - | hyperopt010/011 | :star: T for datasize=32|
| use_lr_mult | T/F | - | hyperopt011 | :no_mouth: F for datasize=32 |
| batch norm | T/F | - | [hyperopt015 vs hyperopt007], hyperopt017 | :yum: T for batch size=4|
| edge_weight | 0.1,0.2,0.5,1.0 | 0.03 | hyperopt016 | :no_mouth: without edge will better |
| momentum | 0.1,0.3,0.5,0.7,0.9 | - | hyperopt018 | :no_mouth: |
| upsample layer | 3,4,5 | - | hyperopt019 | :star: 5 for fcn |
| use_bias | T/F | - | hyperopt020 | :heavy_plus_sign: T for fcn32|

**Comment by yzbx** [comment]
# hyper params experiment summary

no influence-> obviously influence
:no_mouth: < :heavy_plus_sign: < :yum: < :star:
### semantic segmentation miou (mean/max)
| hyper params | value |  miou |  experiments |  others |
| - | - | - | - | - |
| class weight | T/F |  0.36-0.37, 0.35-0.38 | class_weight026/027 | :no_mouth: |
| focal loss gamma | [1,2] | 0.377 - 0.378, 0.386 - 0.403 | focal_loss028/029 | :yum: |

**Comment by yzbx** [comment]
# image classification hyper params
acc (mean/max)

| hyper params | value | acc | experiments | others |
| use_data_aug | T/F | 0.72/0.66 | aug | :star: |
| use_ortho_loss | T/F | - / - | ortho | :heavy_plus_sign: |
| scheduler | poly_pop, rop, none | poly_rop, rop, pop | :heavy_plus_sign: |
